experimental/Schemes/BlowupMorphism.jl

```diff
@@ -497,7 +497,7 @@ end
 function Base.show(io::IO, Bl::BlowupMorphism)
   io = pretty(io)
   if get(io, :supercompact, false)
-    print(io, "Morphism")
+    print(io, "Blow-up morphism")
```

Comment:
Shouldn't this match the detailed printing below? It just uses `"Blow up"`. But perhaps they are intentionally different?

Reply:
Good point. Actually both of them are not as they should be since it should be a unique word. I did not notice that, I will modify it now, thanks!